src/app/[locale]/authorize/page.tsx

```diff
@@ -40,10 +60,10 @@ export default async function AuthorizePage({
         width={40}
         height={40}
       />
-      <Text>{t("authorize.body")}</Text>
+      <Text>{t("authorize.body", { editor: editorName })}</Text>
       <Button
         component="a"
-        href={`vscode://testausserveri-ry.testaustime/authorize?token=${token}`}
+        href={loginUrl}
```

Comment:
This is not correct, `loginUrl` is a path in the frontend, not a path in the extension. You need to create a new map for the extension urls too.